pom.xml

```diff
@@ -70,16 +70,15 @@
         <!-- Not using default port to reduce chances of shutting down an external MongoDB instance -->
         <mongodb.embedded.port>27018</mongodb.embedded.port>
 
-        <!-- JPA 2.2 patch -->
-
-        <jpa22patch.version>1.0.0.Beta2</jpa22patch.version>
-        <jpa22patch.filename>hibernate-jpa-api-2.2-wildflymodules-${jpa22patch.version}-wildfly-${version.wildfly}-patch.zip</jpa22patch.filename>
-
         <!-- Module slots -->
 
         <module-slot.org.hibernate.ogm.short-id>${parsed-version.majorVersion}.${parsed-version.minorVersion}</module-slot.org.hibernate.ogm.short-id>
         <module-slot.org.hibernate.ogm.jipijapa.full-id>${project.version}</module-slot.org.hibernate.ogm.jipijapa.full-id>
-        <module-slot.org.hibernate.search.short-id>${parsed-version.org.hibernate.search.majorVersion}.${parsed-version.org.hibernate.search.minorVersion}</module-slot.org.hibernate.search.short-id>
+
+        <!-- Temporary use of full version *5.10.0.Final* as module slot for Hibernate Search -->
+        <!-- Because *5.1* slot is reserved to *5.10.1.Final* in Wildfly 13.0.0.Final -->
+        <module-slot.org.hibernate.search.short-id>${version.org.hibernate.search}</module-slot.org.hibernate.search.short-id>
```

Comment:
I think, you mean "5.10", not "5.1".

By the way, is there a reason why you can't simply use the 5.10.1 version from WF? Or did we decide to override it anyway?

Wildfly should be written WildFly (with a capital F)

Reply:
Thank you, yes of course it was 5.10. Good question! Unfortunately we cant use 5.10.1 because the **slot name** is defined by the feature pack project: org.hibernate:hibernate-search-jbossmodules-orm and the wildfly-server-provisioning-maven-plugin can't change the slot name. There is a debate on capability to change or define the final module instead at provisioning time.

Reply:
If we decide that we want to override the WF version even if we use the same major.minor, I think we should simply move this to be a `full-id` and always use the full version.

Reply:
I would prefer to use use the *full-id* as a temporary solution. But as you prefer. Using short slot it is in general more easier to the user.

Reply:
Well, is the user supposed to do something with it? I don't know what exactly @Sanne had in mind with the feature packs.

For the user, the easier would be that we just use the version of WF but I'm not sure it's something sustainable (considering it will happen to be in line sometimes but mostly not). The issue here is that, when the versions are aligned, we end up doing some weird gymnastic which is not very desirable either.

@Sanne any idea on how we want to do things here?

Reply:
We decided to use slot names in the format of major.minor when it comes to "depending on another project" as SPIs should be stable enough, so to make it easy for people to drop-in a micro fix without having to change other project modules.

Dependencies across modules of the same project can use the full version. For example hibernate-search-orm would depend on hibernate-search-engine using the full version.

Incidentally, that's why we release aliased modules in feature packs as well.